src/gui/static/src/app/services/wallet.service.ts

```diff
@@ -81,6 +82,22 @@ export class WalletService {
       .flatMap(addresses => this.apiService.get('outputs', {addrs: addresses}));
   }
 
+  outputsWithWallets(): Observable<any> {
+    return Observable.zip(this.all(), this.outputs(), (wallets, outputs) => {
+      wallets = JSON.parse(JSON.stringify(wallets));
```

Comment:
I'm modifying wallets provided by wallet.service; since I don't want to modify the original object, I'm using this "hack" to create a copy of it. Any ideas on how to improve it?